tac/project-updates/2025/2025-annual-Identus.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: 2025 Annual Review Identus
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+LF Decentralized Trust Identus [LFX Insights Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/reports/contributors?project=identus&bestPractice=false&dateFilters=Last%20Year&dateRange=2024-01-01%20to%202024-12-31&compare=PP&granularity=month&hideBots=true&repository=all)
```

Comment:
The LFX Insights numbers are very low compared to other projects: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/best-practice-score?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%2012%20Months&dateRange=2024-01-24%20to%202025-01-23&compare=PP&granularity=month&hideBots=true. I guess this is because only the `governance` repo has been migrated to the new org? Do you have a plan for migration of the rest sometime soon? I don't see any comments about this further below.

Reply:
Also, it looks like you don't have the OpenSSF scorecard configured for Identus: https://scorecard.dev/viewer/?uri=github.com%2Fhyperledger%2Fidentus. Please keep in mind that the OpenSSF Scorecard and the OpenSSF Best Practices badges must be configured for all projects. Soon, TAC project reviews will routinely consider scores from here and from the LFX Insights. For reference on how to configure badges, please see lines 2-3 from the Cacti README file: https://raw.githubusercontent.com/hyperledger-cacti/cacti/refs/heads/main/README.md.